content/en/docs/concepts/policy/limit-range.md

```diff
@@ -1,64 +1,61 @@
 ---
 reviewers:
 - sftim
-title: LimitRange
+- kbhawkey
+title: Limit Ranges
 content_template: templates/concept
 weight: 10
 ---
 
 {{% capture overview %}}
 
-By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on kubernetes cluster.
+By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on a Kubernetes cluster.
 With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
-Within a namespace, a Pod/Container can still consume as much CPU and memory as the defined namespace's resource quota.
-there is a concern that one Pod/Container could monopolise its namespace resources.
+Within a namespace, a Pod or Container can still consume as much CPU and memory as the defined namespace's resource quota.
+there is a concern that one Pod or Container could monopolise its namespace resources.
 LimitRange is a tool to address this concern.

```

Comment:
Possible edits:
Within a namespace, a Pod or Container can consume as much CPU and memory as defined by the namespace's resource quota. There is a concern that one Pod or Container could monopolize all of the resources. Limit Range is a policy to address the allocation of resources in a namespace.

Reply:
update the contents. with some part of the paragraph, not the last sentence.
Thanks